docs/ci-setup.md

```diff
@@ -0,0 +1,167 @@
+# Notes to Setup an SGX VM for CI Testing
+
+# SGX PSW
+
+- install `aesmd`, configure with `default quoting type = ecdsa_256` and the preferred DCAP quote provider lib (with its config)
+
+# containerd
+We pre-install the CoCo containerd and tools with a simple command:
+
+```bash
+curl -fsSL https://github.com/confidential-containers/containerd/releases/download/v1.6.8.1/cri-containerd-cni-1.6.8.1-linux-amd64.tar.gz | sudo tar zx -C /
+```
+
+And it's configured with the following runtimehandler:
+```
+$ tail -3 /etc/containerd/config.toml
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.enclavecc]
+  cri_handler = "cc"
+  runtime_type = "io.containerd.rune.v2"
+```
+
+# verdictd (for EAA-KBC testing)
+
+## Set-up
+
+We provide a verdictd image for quick set-up. Run the following scripts to set up a verdictd server:
+
+```bash
+mkdir -p $VERDICTD_WORKDIR/data
+docker run -d -v /etc/sgx_default_qcnl.conf:/etc/sgx_default_qcnl.conf \
+    -v $VERDICTD_WORKDIR/data:/opt/verdictd \
+    --device /dev/sgx_enclave \
+    --device /dev/sgx_provision \
+    -p 12345:12345 \
+    --env RUST_LOG=debug \
+    xynnn007/verdictd:v0.5.0-rc1 \
```

Comment:
@Xynnn007 did we make `Dockefile`s available for this and kbs/as below?

Reply:
Yes. Verdictd (attention: are in CC branch)
- dockerfile https://github.com/inclavare-containers/verdictd/blob/CC/Dockerfile
- source code: https://github.com/inclavare-containers/verdictd/tree/CC

AS:
- Dockerfile & source code are in AS' repo. As occlum verifier was merged after v0.5.0, I had made a new image `docker.io/xynnn007/attestation-service:sgx-v0.6.0` based on the commit `32d0cbf5baf93566f7819981ec9e40e5c6c9ba55` https://github.com/confidential-containers/attestation-service/commit/32d0cbf5baf93566f7819981ec9e40e5c6c9ba55

Reply:
thanks! I added these links to the doc too.